d3-scripts/tests/__fixtures__/non-existent-parents/test.behaviour.d3.yaml

```diff
@@ -3,4 +3,4 @@ credentialSubject:
   id: aa92ebae-4928-4456-b597-cbd7657e9e83
   ruleName: "Test Behaviour"
   parents:
-    - id: fc4d5a51-f985-4de1-a157-51ea9ca5e9c0
+    - "fc4d5a51-f985-4de1-a157-51ea9ca5e9c0"
```

Comment:
Are the quotes necessary on the guid in this and the following files? I think the YAML parser treats entries as strings by default

Reply:
It appears they are not, I'm sure not having quotes caused an error at some point :thinking:

Reply:
Ah ok, maybe list items parse differently to values like the guide in `id` :+1: